Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1671,10 +1691,21 @@ namespace Multiplayer
         const auto& networkInterfaces = AZ::Interface<AzNetworking::INetworking>::Get()->GetNetworkInterfaces();
         for (const auto& networkInterface : networkInterfaces)
         {
-            if (networkInterface.second->GetType() != ProtocolType::Udp)
+            if (bg_captureTransportType == 0) // look for UDP
             {
-                continue;
+                if (networkInterface.second->GetType() != ProtocolType::Udp)
+                {
+                    continue;
+                }
+            }
+            else // look for TCP
+            {
+                if (networkInterface.second->GetType() != ProtocolType::Tcp)
+                {
+                    continue;
+                }
             }
```

Comment:
This could just condense down to `networkInterface.second->GetType() != bg_captureTransportType`

Reply:
Done.